### PR TITLE
Fix Safari 11 PKCS bug

### LINKED ISSUE #11394: A4A: Crypto Bug
```
Error: Signing service error for google: Failed to import key ({"alg":"RS256","e":"AQAB","ext":true,"kid":"1","kty":"RSA","n":"z43rjaJ9PLk1FHMEL31_ILXGtUTN03rxJ9amD9y3BRDpbTA-GkUKiQM07xAd8OXPUZRqcjvXQfc7b1RCEtwrcfx9oBRdF78QMA4tLLCqSHP0tSuqYF0fA7-GyTFWDcYzey90jRFNNWxjzKrvSazacE0TvJ8S_AVP4EV67VdbByCC1tpBzLhhy7RFHp2cXGTpWYUqZUAVUdJoeBuCho_zQz2au7c6sDaLiF-uYL9Td9MrZ6tSLo3MeMIZia4WgWqjTDICR0h-zlbHUd0K9CoXbGTt5nvkebXHmbKd99ma6zRYVlYNJTuSqsRCBNYtCTFVHIZeBlkjHKsQ46HTZPexZw"}): Member JsonWebKey.kty is required and must be an instance of DOMString
    at message (https://raw.githubusercontent.com/ampproject/amphtml/1506043780363/src/log.js:490:22)
    at https://raw.githubusercontent.com/ampproject/amphtml/1506043780363/src/log.js:220:43
    at https://raw.githubusercontent.com/ampproject/amphtml/1506043780363/src/log.js:234:36
    at https://raw.githubusercontent.com/ampproject/amphtml/1506043780363/extensions/amp-a4a/0.1/signature-verifier.js:445:6
```

Seems to be new to Safari 11.

/cc @ampproject/a4a

### PR DESCRIPTION
**Pull request by taymonbeal**
Safari 11 finally started supporting the modern unprefixed Web Crypto API, but left the old one around for backwards compatibility. This played havoc with our feature detection and caused any attempt to use public-key crypto on Safari 11 (i.e., Fast Fetch preferential rendering) to fail. This fix makes sure that the crypto service always knows which version of the API it's using.

Fixes #11394.

### CODE REVIEW

File: src/service/crypto-impl.js [hunk 1]
```diff
@@ -30,11 +30,22 @@ export class Crypto {
...
+        subtle = win.crypto.subtle;
+      } else if (win.crypto.webkitSubtle) {
+        subtle = win.crypto.webkitSubtle;
+        isLegacyWebkit = true;
+      }
+    }
+
     /** @private @const {?webCrypto.SubtleCrypto} */
-    this.subtle_ = getSubtle(win);
+    this.subtle_ = subtle;
```
Comment (jonkeller) [inline src/service/crypto-impl.js]: Nit: Any reason to not just use this.subtle_ on lines 37, 39?

Comment (taymonbeal) [reply src/service/crypto-impl.js]: Then it couldn't be `@const` anymore.

**Comment by jonkeller** [comment]
Good point. Disregard.

**Comment by taymonbeal** [comment]
Hey, anything blocking a merge at this point?

**Pull request merged**